### PR TITLE
Bug 1114864 - Add Igbo (ig) keyboard layout.

### PR DESCRIPTION
**Pull request by RudyLu**

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=12b4a40b1ed278612570b4a6a50e1d5378aa6dc4

File: apps/keyboard/js/layouts/ig.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Keyboards.ig = {
+  label: ' Igbo',
```
Comment (timdream) [inline apps/keyboard/js/layouts/ig.js]: Extra space.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1907310bf2f29d68ebe79996a871afd258aabd3c

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=25587fbc8d3e414a047b1bbc95298f600fbb212a

**Comment by autolander** [comment]
http://docs.taskcluster.net/tools/task-graph-inspector/#H3Mw6mD4TPqUvcv-cj4EHQ

The pull request failed to pass integration tests. It could not be landed, please try again.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3c35c22117e2cd977c102c0c501dc5033cba5e51

**Pull request merged**